tests/BreadcrumbTest.php

```diff
@@ -102,4 +102,19 @@ public function testWithoutMetadata(): void
         $this->assertSame(['foo' => 'bar'], $breadcrumb->getMetadata());
         $this->assertArrayNotHasKey('foo', $newBreadcrumb->getMetadata());
     }
+
+    public function testBreadcrumbSerialized(): void
+    {
+        $type = Breadcrumb::TYPE_USER;
+        $category = 'foo';
+        $data = ['baz' => 'bar'];
+        $breadcrumb = new Breadcrumb(Breadcrumb::LEVEL_INFO, $type, $category, null, $data);
+        $breadcrumbSerialized = $breadcrumb->jsonSerialize();
+
+        $this->assertEquals($type, $breadcrumbSerialized['type']);
+        $this->assertEquals($category, $breadcrumbSerialized['category']);
+        $this->assertNull($breadcrumbSerialized['message']);
+        $this->assertEquals($data, $breadcrumbSerialized['data']);
+        $this->assertArrayHasKey('timestamp', $breadcrumbSerialized);
```

Comment:
Since this test is marked as 'time-sensitive' we should be able to test against the real value of timestamp. Also probably it makes more sense to compare against the expected result of `jsonSerialize` instead of comparing each field separately

Reply:
Now I compare an expected array with the result of jsonSerialize method